plugins/flutter_aepcore/android/src/main/java/com/adobe/marketing/mobile/flutter/flutter_aepcore/FlutterAEPCorePlugin.java

```diff
@@ -97,6 +104,59 @@ public void onMethodCall(MethodCall call, @NonNull Result result) {
         }
     }
 
+   private void handleInitialize(Result result, Object arguments) {
+       if (!(arguments instanceof Map)) {
+           result.error("INVALID_ARGUMENT", "Initialize failed because arguments is not a Map", null);
+           return;
+       }
+
+       if (application == null) {
+           result.error("INITIALIZATION_ERROR", "Application context is null", null);
+           return;
+       }
+
+       Map initOptionsMap = (Map) arguments;
```

Comment:
Perform a safe cast. 
```
if (arguments instanceof Map) {
  Map initOptionsMap = (Map) arguments;
}
```

I also recommend moving this logic to `FlutterAEPCoreDataBridge.initOptionsFromMap()`